src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Statistics {
+  // aggregated score for a list of comments  
```

Comment:
As discussed offline, maybe also mention what the scores are going to be?
Either "raw scores from range -1 to 1" or "bucketized scores in buckets blablabla"